lib/pbench/agent/tool_data_sink.py

```diff
@@ -21,12 +21,18 @@
 import subprocess
 import sys
 import tempfile
+
+from configparser import DuplicateSectionError
+from daemon import DaemonContext
+from jinja2 import Environment, FileSystemLoader
+from pathlib import Path
```

Comment:
This looks like a bad merge as these imports appear to be repeated below.

Reply:
Ohh,, I must have missed that